src/candidate/CandidateDetailContainer.js

```diff
@@ -0,0 +1,13 @@
+import { connect } from 'react-redux';
+import { toggleFavorite } from './redux/candidates';
+import { getCandidateSummary } from './viewSelectors';
+import CandidateDetail from './CandidateDetail';
+
+const Container = connect(
+  (state, ownProps) => ({
+    summary: getCandidateSummary(state, ownProps.candidateId),
+  }),
+  { toggleFavorite },
+)(CandidateDetail);
+
+export default Container;
```

Comment:
My personal style is to place the redux connected component in the same file alongside the plain react component. React-redux provides a property called `BaseComponent` that makes it very easy to access the plain component in unit tests or other cases where you don't want redux